lib/parsers/webhook_parser.rb

```diff
@@ -70,6 +76,8 @@ def branch
         when 'bitbucket'
           return @data['push']['changes'][0]['new']['name'] unless deleted?
           @data['push']['changes'][0]['old']['name']
+        when 'bitbucketserver'
+          @data['changes'][0]['refId'].sub('refs/heads/', '')
```

Comment:
This should probably replace the `stash` parser as we, despite the name, only support the `bitbucket server` webhook format.

Reply:
I *was* using the stash parser with stash...